test/1_stdlib/TestUserInfo.swift

```diff
@@ -59,8 +59,8 @@ class TestUserInfo : TestUserInfoSuper {
         let userInfoKey = "userInfoKey"
         let notifName = Notification.Name(rawValue: "TestSwiftNotification")
         let testStructure = SomeStructure(i: 5, str: "10", sub: SubStruct(i: 6, str: "11"))
-        let info: [String : Any] = [
-            userInfoKey : testStructure
+        let info: [AnyHashable : Any] = [
+            AnyHashable(userInfoKey) : testStructure
```

Comment:
Yea it is fallout from the AnyHashable being a structure and not a protocol (and not being transparently convertible to the counterpart